.circleci/config.yml

```diff
@@ -1,6 +1,4 @@
 version: 2.1
-orbs:
-  snyk: snyk/snyk@2.0.1
```

Comment:
do we need to provide a new orb here? or will this use a default one?

Reply:
maybe this? 
```yaml
orbs:
  golang: circleci/go@1.10.0
```

Reply:
actually we won't use Snyk at all anymore and i'm not aware of a new static code analysis tool validated yet.